app/uk.gov.hmrc.helptosaveproxy/controllers/HelpToSaveController.scala

```diff
@@ -69,6 +73,18 @@ class CreateAccountController @Inject() (nsiConnector:                NSIConnect
       }
   }
 
+  def getAccountByNino(nino: NINO, version: String, systemId: String, correlationId: UUID): Action[AnyContent] = Action.async { implicit request ⇒
```

Comment:
This makes correlationId a required parameter doesn't it? It would be nice if it was optional like it is in the NS&I API.

Also it might be nice to pass through all parameters so that if NS&I add support for a new parameter then this microservice doesn't need to be changed. `request.rawQueryString` or `request.queryString` could be used for this.

Reply:
yeah correlationId is optional , but there is no reason why we cant have one because we can always generate one and send it to NSI, that is the reason why i made it non-optional . 
If you still want to make it optional , i can do that , no issues